.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+      - name: Install dependencies
+        run: dotnet restore -p:NuGetBuild=True
+      - name: Build
+        run: dotnet build --configuration Release --no-restore  -p:NuGetBuild=True
+      - name: Test
+        run: dotnet test --no-restore  -p:NuGetBuild=True
```

Comment:
It's a noop if there are no test projects found, so shouldn't take any extra time really. just useful perhaps in case you one day add a test project to something I guess?

Reply:
We have unit tests in Helpful Libraries, and UI tests in the UI Testing Toolbox. Due to the latter, i.e. that projects can have UI tests that we don't want to execute, I'd remove this.

Reply:
Sure

Reply:
Done